modyn/protos/README.md

```diff
@@ -2,10 +2,6 @@
 
 This assumes python 3.6+ is installed.
 
-## Install the grpcio-tools package
-
-`pip install grpcio-tools`
-
 ## Generate the python files
```

Comment:
we already installed `grpcio-tools` in `dev-requirements.txt`. And if the developer follows the README.md, she/he already did `pip install -r dev-requirements.txt` as README.md says.